docs/generate-features.md

```diff
@@ -0,0 +1,44 @@
+#### generate-features
+---
+Scan the class files of your application and create a file containing the Liberty `features` that it requires.
+
+This goal is available as a tech preview in the 3.5.2-SNAPSHOT. Please provide feedback by opening an issue at https://github.com/OpenLiberty/ci.maven.
+
+Lifecycle
+
+This goal is not part of the Maven lifecycle so to use it in your build you will need to understand its dependencies. Since it will scan the class files of your application it must be run after you have run the compile goal. The list of features that it generates will be used by the `liberty:create` and the `liberty:install-feature` goals so run this goal first.
```

Comment:
Quotes around compile goal and consider rewording
```
Since it will scan the class files of your application it must be run after the `compile` goal
```

Reply:
At first I was keeping it generic between Maven and Gradle but it is more correct this way.